androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/PlayerControls.kt

```diff
@@ -0,0 +1,96 @@
+package com.profusion.androidenhancedvideoplayer.components
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import com.profusion.androidenhancedvideoplayer.R
+
+@Composable
+fun PlayerControls(
+    isPlaying: Boolean,
+    hasEnded: Boolean,
+    onPreviousClick: () -> Unit,
+    onPauseToggle: () -> Unit,
+    onNextClick: () -> Unit,
+    decoration: ControlsDecoration,
+    modifier: Modifier = Modifier
+) {
+    Box(
+        contentAlignment = Alignment.CenterStart,
+        modifier = modifier
+            .background(decoration.backgroundColor)
+            .fillMaxSize()
+    ) {
+        Row(
+            horizontalArrangement = Arrangement.SpaceEvenly,
+            modifier = Modifier
+                .align(Alignment.Center)
+                .fillMaxWidth()
+        ) {
+            IconButton(onClick = onPreviousClick) {
+                decoration.previousIcon()
+            }
+            IconButton(onClick = onPauseToggle) {
+                decoration.playOrPauseIcon(
+                    isPlaying,
+                    hasEnded
+                )
+            }
+            IconButton(onClick = onNextClick) {
+                decoration.nextIcon()
+            }
+        }
+    }
+}
+
+@Composable
+fun DefaultPreviousIcon() {
```

Comment:
I think we will use a lot of icons, maybe it is better to move all icons to a single file/folder? like `utils/PlayerIcons.kt` containing all of the composables